.gitignore

```diff
@@ -29,4 +29,3 @@ cmake-build-*/
 *.swp
 *.swo
 
-*.e
```

Comment:
Can we keep .gold.e in the case of if we update any gold exodus files? These are used as the baseline for some regressions tests

Reply:
Additionally in MPI mode the generated files have the following patterns:
*.e.<num ranks>.<rank>

e.g.
```
two_body_contact.mpi.e.4.0
two_body_contact.mpi.e.4.1
two_body_contact.mpi.e.4.2
two_body_contact.mpi.e.4.3
```

Reply:
Never mind -- misread the diff